### DISCUSSION TITLE
How to specify adbkey and adbkey.pub

### DISCUSSION THREAD

**Discussion by Boris-code** [yume-chan/ya-webadb#636] [Q&A] [answered]
How to specify adbkey and adbkey.pub

**Comment by yume-chan** [comment, marked as answer]
https://docs.tangoapp.dev/tango/daemon/credential-store/

**Comment by Boris-code** [comment]
I have adbkey and adbkey.pub file，i want to use myself key，not generate

**Reply by yume-chan** [reply]
It's in the "Node.js" tab
